### PR TITLE
SCC-4066 - Search results VQA Pt. 2

### PR DESCRIPTION
**Pull request by dgcohen**
Some more visual changes based on the last few items in the [VQA doc](https://docs.google.com/document/d/1qgHzrubMwxx_UvVu_hfVhDfbPEPPbgrypHv3BohSPtI/edit#heading=h.wq30kg5jzx41).

Specifically:

1. Search Results Card - Multiple Items + Available Online Links
2. DRB Sidebar
3. Pagination
4. Refine Search Dropdown

### CODE REVIEW

File: pages/search/index.tsx [hunk 1]
```diff
@@ -161,18 +161,21 @@ export default function Search({
...
             />
           </>
+        ) : /**
+         * TODO: The logic and copy for different scenarios will need to be added when
+         * filters are implemented
+         */
+        isLoading ? (
+          <SkeletonLoader showImage={false} />
         ) : (
-          /**
```
Comment (dgcohen) [inline pages/search/index.tsx:172]: This is not related to VQA but a small bug fix I noticed in passing. Fixes an issue where there was no loading state after searching from a no results page. Let me know if you'd like me to move this to a separate PR.

File: styles/components/Search.module.scss [hunk 2]
```diff
@@ -3,6 +3,11 @@
 
 .filterTags {
   margin-bottom: var(--nypl-space-xs);
+
```
Comment (dgcohen) [inline styles/components/Search.module.scss:6]: This is a DS styling override, maybe we should see whether this should be implemented there?

**Comment by charmingduchess** [comment]
I don't know exactly where the styling for this is, but I'm seeing a larger space between rows in the single result card than is specified in the vqa doc.

**Comment by dgcohen** [comment]
> I don't know exactly where the styling for this is, but I'm seeing a larger space between rows in the single result card than is specified in the vqa doc. Also, the space between DRB title and read online button looks smaller than the VQA doc.

I believe the single result VQA fixes are part of Edwin's PR (pt.1 ). Not sure I'm seeing the same issue with the DRB title spacing, looks like 24px on my local and vercel.

**Comment by charmingduchess** [comment]
I think the thing with the spacing on DRB results is a thing when there isn't an author listed, which isn't a scenario outlined in the doc. If Apoorva doesn't flag it I guess it's not a big deal. 
Screenshot 2024-04-12 at 12 17 24 PM

**Review by charmingduchess — APPROVED** [review]
LGTM 🔥

File: src/components/DRB/DRBCard.tsx [hunk 3]
```diff
@@ -21,29 +22,33 @@ const DRBCard = ({ drbResult }: DRBCardProps) => {
   if (!drbResult) return null
 
   return (
-    <Card backgroundColor="ui.bg.default" p="xs">
+    <Card backgroundColor="white" p="s" borderRadius="5px">
```
Comment (EdwinGuzman) [inline src/components/DRB/DRBCard.tsx:25]: Use `ui.white`.

File: src/components/SearchResult/ElectronicResourcesLink.tsx [hunk 4]
```diff
@@ -22,23 +18,30 @@ const ElectronicResourcesLink = ({
...
+          isUnderlined={false}
         >
           {electronicResources[0].prefLabel || electronicResources[0].url}
         </DSLink>
       ) : (
         <RCLink
           href={`${BASE_URL}${bibUrl}#electronic-resources`}
           type="standalone"
+          fontSize="desktop.body.body2"
+          fontWeight="bold"
```
Comment (EdwinGuzman) [inline src/components/SearchResult/ElectronicResourcesLink.tsx:41]: This should be medium.

File: src/components/DRB/DRBContainer.tsx [hunk 5]
```diff
@@ -33,8 +33,8 @@ const DRBContainer = ({
   const drbQuery = getDRBQueryStringFromSearchParams(searchParams)
 
   return (
-    <Card isBordered>
-      <CardHeading level="h3">
+    <Card backgroundColor="ui.bg.default" p="s" borderRadius="5px">
+      <CardHeading level="h6">
```
Comment (EdwinGuzman) [inline src/components/DRB/DRBContainer.tsx:37]: I think this should be an h2 to be on the same level as the "Displaying 1-50 of ...." text. The VQA doc says it's in the _style_ of h6 so do `size="h6"` instead.

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Some minor updates.

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**